src/Compilers/CSharp/Portable/Symbols/Source/SourceNamespaceSymbol.cs

```diff
@@ -400,14 +400,14 @@ private static void CheckMembers(NamespaceSymbol @namespace, Dictionary<string,
                 }
             }
 
-            static bool isFileTypeInSeparateFileFrom(SourceNamedTypeSymbol possibleFileType, SourceNamedTypeSymbol otherSymbol)
+            static bool isFileLocalTypeInSeparateFileFrom(SourceNamedTypeSymbol possibleFileLocalType, SourceNamedTypeSymbol otherSymbol)
             {
-                if (!possibleFileType.IsFile)
+                if (!possibleFileLocalType.IsFile)
```

Comment:
> .IsFile

Is there a missing rename here?

https://github.com/dotnet/roslyn/blob/7fc34f9e211bef9499cd1bdf6a11630f12052de1/src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs#L829

I see the implementation is about the modifier, but I think the API itself is meant to be about the language concept?